code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -2084,4 +2084,171 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	taste_description = "a sweet and sour slice of apple"
 	glass_icon_state = "jack_rose"
 	glass_name = "Jack Rose"
-	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
\ No newline at end of file
+	glass_desc = "Enough of these, and you really will start to suppose your toeses are roses."
+
+/datum/reagent/consumable/ethanol/turbo
+	name = "Turbo"
+	id = "turbo"
+	description = "A turbulent cocktail associated with outlaw hoverbike racing. Not for the faint of heart."
+	color = "#e94c3a"
+	boozepwr = 85
+	quality = DRINK_VERYGOOD
+	taste_description = "the outlaw spirit"
+	glass_icon_state = "turbo"
+	glass_name = "Turbo"
+	glass_desc = "A turbulent cocktail for outlaw hoverbikers."
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_add(mob/living/M)
+	..()
+	M.add_movespeed_modifier(id, update=TRUE, priority=100, multiplicative_slowdown=-0.20, blacklisted_movetypes=(FLYING|FLOATING))
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_delete(mob/living/M)
+	M.remove_movespeed_modifier(id)
+	..()
+
+
+/datum/reagent/consumable/ethanol/turbo/on_mob_life(mob/living/carbon/M)
+	if(prob(7))
+		to_chat(M, "<span class='notice'>[pick("You feel disregard for the rule of law.", "You want to go fast.", "You feel pumped!", "Your head is pounding.", "Your thoughts are racing..")]</span>")
+	M.adjustStaminaLoss(-8)
+	return ..()
+
+/datum/reagent/consumable/ethanol/old_timer
+	name = "Old Timer"
+	id = "old_timer"
+	description = "An archaic potation enjoyed by old coots of all ages."
+	color = "#996835"
+	boozepwr = 35
+	quality = DRINK_NICE
+	taste_description = "simpler times"
+	glass_icon_state = "old_timer"
+	glass_name = "Old Timer"
+	glass_desc = "WARNING! May cause premature aging!"
+
+/datum/reagent/consumable/ethanol/old_timer/on_mob_life(mob/living/carbon/M)
+	if(prob(20))
```

Comment:
Maybe make this a 100% chance per tick instead of a 20% chance one so that sec/antags can more easily forcibly age people as a form of punishment?

Reply:
I want people to be able to enjoy like a half glass without it turning their hair grey. I will consider increasing the rate of age gain though.